docs/src/tutorials/device-monitoring.md

```diff
@@ -16,6 +16,14 @@ Enabling monitoring on your device is a 3-steps process:
 2. [Configure `collectd`](#configure-collectd),
 3. [Enable thin-edge.io monitoring](#enable-thin-edge-monitoring).
 
+## Install `mosquitto` client library
+collectd depends on mosquitto client library.
+So, need to install libmosquitto.
```

Comment:
This is not correct statement, it is not `collectd` depending on the `libmosquitto` but rather the plugin we use for MQTT communication.

Reply:
But the plugin is part of the collectd right?. As a whole its collectd from thin-edge pov.

Reply:
Making explicit that this is the MQTT plugin that requires the library makes clearer the need for an install (otherwise one can see this as a dependency bug of the collectd debian package).

```suggestion
Since thin-edge.io uses the MQTT plugin of `collectd`, one needs to install the mosquitto client library (either `libmosquitto1` or `mosquitto-clients`.
```